src/lib/utils/translations.ts

```diff
@@ -210,3 +212,19 @@ const getRequiredNamespacesForLayout = (layout?: string) => {
 
   return requiredNamespaces
 }
+
+/**
+ * Checks if the primary namespace .json file exists for the given locale
+ * @param locale Path locale
+ * @param primaryNs Primary namespace for the page
+ * @param existsSync Passed from fs library
+ * @returns false if the primary namespace .json file exists, true otherwise
+ */
+export const getIsContentNotTranslated = (
```

Comment:
I'd prefer a positive naming always. Improves the reading experience for devs. Then you can do the negative on the consumer's side

```ts
const isContentNotTranslated = !existsNamespace(...)

// or even better
const isContentTranslated = existsNamespace(...)

if (!isContentTranslated) {
  ...
}
```

Reading this function name without any context makes it difficult to understand its purpose.

And we're returning a boolean, no need the `get` prefix on the name.

```suggestion
export const existsNamespace = (
```

Reply:
Agree... had named to match the existing boolean name we were using (`contentNotTranslateds`), but can update this